.github/ISSUE_TEMPLATE/backend-checklist.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
@MV88 Thank you for this but some of the info you put here are redundant and I also think it is better to remove this template for now and open a dedicated issue for exchanging about this and so fine the best way for address needs we have across different releases. 
@offtherailz however, I would also say that, waiting for the above exchange, it would be good to have a new point in the release issue for checking the project templates alignment, if it doesn't exists yet.